Night CI 2023-07-27: Failed

action url: https://github.com/spidernet-io/spiderpool/actions/runs/5684502180

```
CILIUM_HELM_OPTIONS:  --set cni.exclusive=false                             --set kubeProxyReplacement=disabled                             --set k8sServiceHost=spiderpool0727191431-control-plane                             --set k8sServicePort=6443                             --set bpf.vlanBypass=0  --set ipam.operator.clusterPoolIPv4PodCIDR=10.244.64.0/18                                    --set ipam.operator.clusterPoolIPv6PodCIDR=fd00:10:244::/112                                    --set ipv4.enabled=true                                    --set ipv6.enabled=true        --set image.repository=quay.io/cilium/cilium       --set image.useDigest=false       --set certgen.image.repository=quay.io/cilium/certgen       --set hubble.relay.image.repository=quay.io/cilium/hubble-relay       --set hubble.relay.image.useDigest=false       --set hubble.ui.backend.image.repository=quay.io/cilium/hubble-ui-backend       --set hubble.ui.frontend.image.repository=quay.io/cilium/hubble-ui       --set etcd.image.repository=quay.io/cilium/cilium-etcd-operator       --set operator.image.repository=quay.io/cilium/operator        --set operator.image.useDigest=false        --set preflight.image.repository=quay.io/cilium/cilium       --set preflight.image.useDigest=false       --set nodeinit.image.repository=quay.io/cilium/startup-script 
"cilium" has been added to your repositories
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "cilium" chart repository
Update Complete. ⎈Happy Helming!⎈
Error: execution error at (cilium/templates/cilium-configmap.yaml:884:8): Value ipam.operator.clusterPoolIPv4PodCIDR removed, use ipam.operator.clusterPoolIPv4PodCIDRList instead

Use --debug flag to render out invalid YAML
make[2]: *** [Makefile:23: kind-init] Error 1
make[2]: Leaving directory '/home/runner/work/spiderpool/spiderpool/test'
make[1]: *** [Makefile:298: e2e_init] Error 2
make[1]: Leaving directory '/home/runner/work/spiderpool/spiderpool'
make: *** [Makefile:323: e2e_init_cilium] Error 2
```

--- a/test/scripts/install-default-cni.sh
+++ b/test/scripts/install-default-cni.sh
@@ -122,12 +122,12 @@ function install_cilium() {
                             --set bpf.vlanBypass=0 "
       case ${E2E_IP_FAMILY} in
         ipv4)
-            CILIUM_HELM_OPTIONS+=" --set ipam.operator.clusterPoolIPv4PodCIDR=${CILIUM_CLUSTER_POD_SUBNET_V4} \
+            CILIUM_HELM_OPTIONS+=" --set ipam.operator.clusterPoolIPv4PodCIDRList=${CILIUM_CLUSTER_POD_SUBNET_V4} \
                                    --set ipv4.enabled=true \
                                    --set ipv6.enabled=false "
           ;;
         ipv6)
-            CILIUM_HELM_OPTIONS+=" --set ipam.operator.clusterPoolIPv6PodCIDR=${CILIUM_CLUSTER_POD_SUBNET_V6} \
+            CILIUM_HELM_OPTIONS+=" --set ipam.operator.clusterPoolIPv6PodCIDRList=${CILIUM_CLUSTER_POD_SUBNET_V6} \
                                    --set ipv4.enabled=false \
                                    --set ipv6.enabled=true \
                                    --set tunnel=disabled \
@@ -136,8 +136,8 @@ function install_cilium() {
                                    --set enableIPv6Masquerade=true "
           ;;
         dual)
-            CILIUM_HELM_OPTIONS+=" --set ipam.operator.clusterPoolIPv4PodCIDR=${CILIUM_CLUSTER_POD_SUBNET_V4} \
-                                   --set ipam.operator.clusterPoolIPv6PodCIDR=${CILIUM_CLUSTER_POD_SUBNET_V6} \
+            CILIUM_HELM_OPTIONS+=" --set ipam.operator.clusterPoolIPv4PodCIDRList=${CILIUM_CLUSTER_POD_SUBNET_V4} \
+                                   --set ipam.operator.clusterPoolIPv6PodCIDRList=${CILIUM_CLUSTER_POD_SUBNET_V6} \
                                    --set ipv4.enabled=true \
                                    --set ipv6.enabled=true "
           ;;

